plugin/plugins/mijia/mijia_api/domain/models.py

```
@@ -0,0 +1,203 @@
+        if self.type == PropertyType.INT and type(value) is not int:
+            return False
+        # UINT 需要 >= 0 且排除 bool
+        if self.type == PropertyType.UINT:
+            if type(value) is not int or value < 0:
+                return False
+        if self.type == PropertyType.FLOAT and not isinstance(value, (int, float)):
+            return False
+        if self.type == PropertyType.STRING and not isinstance(value, str):
+            return False
+
+        # 范围检查（支持 [min, max] 和 [min, max, step] 两种格式）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)